### PR TITLE
feat: depend on mosaic 1.0.0

### PR DESCRIPTION
**Pull request by marcelltoth**
Nice to have for #141

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +39,8 @@
   },
   "peerDependencies": {
     "@stoplight/markdown-viewer": "^5.0.0-beta.5",
-    "@stoplight/mosaic": "^1.0.0-beta.59",
+    "@stoplight/mosaic": "^1.0.0",
+    "@stoplight/mosaic-code-viewer": "^1.0.0",
```
Comment (marcelltoth) [inline package.json:43]: This was missing. Needed for (peer of) MDV.

**Review by mallachari — APPROVED** [review]

**Comment by mallachari** [comment]
Hmm, what's wring with those types? Is it some react incompatibility?

**Comment by paulatulis** [comment]
Should the test_and_release workflow pass or is that something that can be ignored?

Also, is there somewhere I could see this in action? If not, since it's a nice-to-have, I will defer to @mallachari's review since he is probably more familiar with what this affects.

**Comment by marcelltoth** [comment]
> 
> 
> Should the test_and_release workflow pass or is that something that can be ignored?

It absolutely should, updated now!

> Also, is there somewhere I could see this in action? If not, since it's a nice-to-have, I will defer to @mallachari's review since he is probably more familiar with what this affects.

Unfortunately no. JSV doesn't have PR environments AFAIK so you have to run it locally. Also there should be no visible change compared to current `beta`.

**Pull request merged**